dbms/src/Storages/DeltaMerge/DeltaMergeStore.cpp

```diff
@@ -1497,7 +1495,7 @@ void DeltaMergeStore::checkSegmentUpdate(const DMContextPtr & dm_context, const
         {
             delta_last_try_flush_rows = delta_rows;
             delta_last_try_flush_bytes = delta_bytes;
-            LOG_FMT_DEBUG(log, "Foreground flush cache {}", segment->info());
+            LOG_FMT_DEBUG(log, "Foreground flush cache in checkSegmentUpdate, thread={} segment={}", thread_type, segment->info());
```

Comment:
```suggestion
            LOG_FMT_INFO(log, "Foreground flush cache in checkSegmentUpdate, thread={} segment={}", toString(thread_type), segment->info());
```
I think we can use the info level here for foreground task

Reply:
Good suggestion! I will have another PR carrying write stall info in the logs, and I can apply the change there.